bitnami/flux/templates/crds/helm-controller/helm.toolkit.fluxcd.io_helmreleases.yaml

```diff
@@ -1,3 +1,4 @@
+# Taken from https://github.com/fluxcd/flux2/releases/download/v2.2.2/install.yaml
```

Comment:
At the moment we will have two kinds of headers:
- Automated CRDs:
```
# Source: <URL>
# Version: <appVersion>
```
- Non-automated CRDs:
```
# Taken from <URL>
```

In this case, Flux does not support automatic updated because its source is an 'install.yaml' file that contains other files, so it has to be performed manually

Reply:
Ahh got it now, thank you for the clarification!